kratos/sources/model_part_io.cpp

```diff
@@ -2053,13 +2054,9 @@ void ModelPartIO::ReadNodalDataBlock(ModelPart& rThisModelPart)
 
     VariablesList rThisVariables = rThisModelPart.GetNodalSolutionStepVariablesList();
 
-
-    if(KratosComponents<Flags >::Has(variable_name))
-    {
+    if(KratosComponents<Flags >::Has(variable_name)) {
```

Comment:
could yo move it at the Matrix, Vector, VariableData, etc...?

Reply:
I don't understand

Reply:
I would expect the "simple" variables at the begining of the if, and the "complex" at the end, its like that all over Kratos (int,double,arrays,vector,matrices,etc...)

Reply:
But the flags was already there, take a look, in here I am  just correcting the {} style

Reply:
And Flags are IMHO the simplest ones